index.html

```diff
@@ -57,6 +57,132 @@ <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
                 </li>
             </ul>
         </section>
+
+        <section class="works wrapper">
+            <h2 class="sr-only">Works Section</h2>
+            <article class="card">
+                <img src="./images/Tonic.png" alt="" aria-hidden="true">
+                <h3 class="title color-caption">Tonic</h3>
+                <div class="profile">
+                    <div class="client">Canopy</div>
+                    <div class="role color-gray">Back End Dev</div>
+                    <div class="year color-gray">2015</div>
+                </div>
+                <p class="details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="tags">
+                    <p class="tag">html</p>
+                    <p class="tag">css</p>
+                    <p class="tag">javascript</p>
+                </div>
```

Comment:
Kindly use `ul` tag instead of `div` tag, and `li` tag instead of `p` tag, and make the same for every project technologies in every project. To improve your structure and semantics.